npc/quests/quests_prontera.txt

```diff
@@ -44,6 +44,8 @@
 //= 2.7 Corrected 'IProntera' typo. (bugreport:4522) [L0ne_W0lf]
 //= 2.8 Updated RE/Pre-RE EXP. [Euphy]
 //= 2.9 Updated Nameless Island addition and added questlog support. [Euphy]
+//= 3.0 Reverted custom/unofficial quest log support for: [Everade]
+//=     - Geoborg Family Curse Quest (Nameless Island addition)
```

Comment:
```suggestion
//= 3.0 Reverted 2.9 by commenting [Everade]
```

Reply:
Not the entire 2.9 was reverted. And i think we would lose essential information on the changelog like this.